app/views/candidate_interface/personal_details/edit.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :title, page_title('personal_details') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <fieldset class="govuk-fieldset">
+      <legend
+        class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8"
+      >
+        <h1 class="govuk-fieldset__heading">
+          Personal details
+        </h1>
+      </legend>
+
+      <%= form_with model: @personal_details_form,
+        url: candidate_interface_personal_details_update_path,
+        builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+        <%= f.govuk_error_summary %>
+
+        <%= f.govuk_text_field :first_name,
+          label: { text: 'First name', size: 'm' },
+          hint_text: 'Or given names' %>
+
+        <%= f.govuk_text_field :last_name,
+          label: { text: 'Last name', size: 'm' },
+          hint_text: 'Or family name' %>
+
+        <%= f.govuk_date_field :date_of_birth,
+          date_of_birth: true,
+          legend: { text: 'Date of birth', tag: 'span' },
+          hint_text: 'For example, 31 3 1980' %>
+
+        <%= f.govuk_text_field :nationality,
+          label: { text: 'Nationality', size: 'm' } %>
+
+        <%= f.govuk_collection_radio_buttons :english_main,
+          [
+            OpenStruct.new(id: 'yes', name: 'Yes'),
+            OpenStruct.new(id: 'no', name: 'No')
+          ],
```

Comment:
Probably should pull this out into the `PersonalDetailsForm` model; `english_main` could be an enum:

```ruby
  enum english_main: {
    yes: "yes",
    no: "no"
  }
```

And then we can probably call `@personal_details_form.english_mains` here and somehow rewrite the next two params. If this isn't possible, as the least we can pull this array out into the form model. 👍